server/app/lib/fusor/resources/customer_portal.rb

```diff
@@ -83,6 +83,7 @@ def self.rest_client(path, credentials)
             options[:headers] = self.default_headers
             options[:user] = credentials[:username]
             options[:password] = credentials[:password]
+            options[:verify_ssl] = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Yes, this was intentional, but it's not required now.

Originally we were testing with a newer rbovirt which required rest_client 1.7.0.
If using rest_client 1.7.0 we needed to be explicit with this :verify_ssl being set to None.

The behavior with 1.6.0 was to default to None, but 1.7.0 defaulted to verifying the CA cert which we aren't configuring.

We can leave this in to be more explicit with what we are doing now.  Yet it's not currently required since our plan with rbovirt is to patch the existing version, so we don't need to upgrade rest_client 1.6.0